src/index.ts

```diff
@@ -18,7 +18,7 @@ export const laboratoryDashboardLink = getSyncLifecycle(
   createHomeDashboardLink({
     name: 'laboratory',
     slot: 'laboratory-dashboard-slot',
```

Comment:
This is the canonical way to translate the dashboard link text:

```suggestion
  // t('laboratory', 'Laboratory')
  createHomeDashboardLink({
    name: 'laboratory',
    slot: 'laboratory-dashboard-slot',
    title: 'Laboratory'
```